packages/main/src/Button.ts

```diff
@@ -246,6 +247,18 @@ class Button extends UI5Element implements IFormElement, IButton {
 	@property({ type: ButtonType, defaultValue: ButtonType.Button })
 	type!: `${ButtonType}`;
 
+	/**
+	 * Describes the accessibility role of the button.
+	 *
+	 * NOTE: Use link role only with a press handler, which performs a navigation. In all other scenarios the default button semantics is recommended.
+	 *
+	 * @default "Button"
+	 * @public
+	 * @since 1.23
+	 */
+	@property({ type: ButtonAccessibleRole, defaultValue: ButtonAccessibleRole.Button })
+	accessibleRole!: string;
```

Comment:
The type here should be `${ButtonAccessibleRole}` by convention. Same way as the above property `type`.